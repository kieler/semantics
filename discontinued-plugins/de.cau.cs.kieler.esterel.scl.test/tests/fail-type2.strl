module fail_type2:
type integer; % Redefinition of predefined type

nothing

end module
