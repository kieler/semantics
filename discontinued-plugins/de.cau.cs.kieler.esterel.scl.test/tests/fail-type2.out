2:redeclaration of type integer
